robotbase/src/main/java/com/team2357/frc2022/Constants.java

```diff
@@ -113,13 +109,15 @@
          */
         public static final double P_DRIVE_VEL = 0;
 
-        /**
-         * Controls if Ggyro is reversed or not.
-         */
-
+        public static final FalconTrajectoryDriveSubsystem.Configuration GET_FALCON_DRIVE_CONFIG() {
+            FalconTrajectoryDriveSubsystem.Configuration config = new FalconTrajectoryDriveSubsystem.Configuration();
+            config.m_isRightInverted = true;
+            config.m_isGyroReversed = true;
+            return config;
+        }
     }
 
-    public final class LIMELIGHT {
+    public static final class LIMELIGHT {
         /** Angle of the Limelight axis from horizontal (degrees) */
         public static final double MOUNTING_ANGLE = 0;
```

Comment:
You need to remove these unused constants, and move their respective comments to the ```GET_LimelightSubsystem_CONFIG```.

Reply:
fixed